fog/distribution/src/main.rs

```diff
@@ -563,8 +568,14 @@ fn build_tx(
     // Sanity
     assert_eq!(utxos_with_proofs.len(), rings.len());
 
+    // This max occurs because the bootstrapped ledger has block version 0,
```

Comment:
Really? Our bootstrap block version is different than all the other blocks?

Reply:
I thought it was true in mainnet and testnet also, I think the bootstrap program used block version zero for the blocks it made. Maybe code changed between now and then

Reply:
Yeah, looks like that is how the code is right now. I forgot about that.